src/lib.rs

```diff
@@ -73,6 +73,12 @@ pub struct TproxyState {
     pub(crate) restore_ipv4_route: Option<Vec<String>>,
     #[cfg(target_os = "linux")]
     pub(crate) restore_ipv6_route: Option<Vec<String>>,
+    #[cfg(target_os = "linux")]
+    pub(crate) restore_gateway_mode: Option<Vec<String>>,
+    #[cfg(target_os = "linux")]
+    pub(crate) restore_port_forwarding: bool,
```

Comment:
Can we call this `restore_ip_forwarding`? Port forwarding usually refers to something else.

Reply:
Yes, You're right.